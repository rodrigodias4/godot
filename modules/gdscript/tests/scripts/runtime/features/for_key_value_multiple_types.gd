func test():
    for u, v in {"key1" : "value1", 2 : "value2", "key3" : 3, 4 : 4, 5.01 : 5.01}:
        print(u, " ", v)
