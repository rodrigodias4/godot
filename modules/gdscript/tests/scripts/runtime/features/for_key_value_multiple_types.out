GDTEST_OK
key1 value1
2 value2
key3 3
4 4
5.01 5.01
